packages/remix/src/build.ts

```diff
@@ -334,6 +334,7 @@ async function createRenderNodeFunction(
     shouldAddSourcemapSupport: false,
     operationType: 'SSR',
     experimentalResponseStreaming: true,
+    framework: 'remix',
```

Comment:
Should this also be added to `@vercel/static-build` which is where most frameworks are used?

I'll defer to @EndangeredMassa who worked on a similar feature recently.